nautobot/docs/development/getting-started.md

```diff
@@ -178,7 +178,23 @@ Additional useful commands for the development environment:
 - `invoke stop` - Stops all containers created by `invoke start`
 
 !!! info
-    By default the Nautobot docker images for development are built with Python 3.7. If you wish to build them with a different Python version, stop any running containers, then set the environment variable `PYTHON_VER` to the desired version (for example, `export PYTHON_VER=3.8`) and rerun the `invoke build` command. As long as `PYTHON_VER` remains set, all other `invoke` tasks will use the containers built for this version instead of the default.
+    By default the Nautobot docker images for development are built with Python 3.7. If you wish to build them with a different Python version, stop any running containers, then set the environment variable `INVOKE_NAUTOBOT_PYTHON_VER` to the desired version (for example, `export INVOKE_NAUTOBOT_PYTHON_VER=3.8`) and rerun the `invoke build` command. As long as `INVOKE_NAUTOBOT_PYTHON_VER` remains set, all other `invoke` tasks will use the containers built for this version instead of the default. See the [invoke configuration](#invoke-configuration) for more information.
```

Comment:
Since you've added the following section, this info callout is maybe obsolete/redundant now? Consider removing it.